.codeclimate.yml

```diff
@@ -11,9 +11,12 @@ engines:
         javascript:
   eslint:
     enabled: true
+    config:
+      config: interface/eslint.conf.js
```

Comment:
It does seem weird. I copied the syntax from:
https://docs.codeclimate.com/docs/eslint
The first **config** may include other options, such as "extensions", the second **config** is used to specify the configuration file's path.